sympy/sets/sets.py

```diff
@@ -652,6 +716,18 @@ def _boundary(self):
     def _measure(self):
         raise NotImplementedError("(%s)._measure" % self)
 
+    def _kind(self):
+        return SetKind(UndefinedKind)
+
+    @staticmethod
+    def _helper_kind(args):
```

Comment:
It's generally recommended to use `classmethod` rather than `staticmethod`.

Reply:
But calling `args` with help of `cls.args` was not possible as args is class property, `args` can only be called using `self.args`. So I was not able to find a way to make it into `classmethod`.
It throws typeError when using classmethod
```
    721         if not cls.args:
    722             return SetKind()
--> 723         elif all(i.kind == cls.args[0].kind for i in cls.args):
    724             return SetKind(cls.args[0].kind)
    725         else:

TypeError: 'property' object is not iterable
```

Reply:
The `args` are being passed in by the caller.

Reply:
It's just
```
    @classmethod
    def _helper_kind(cls, args):

```